scripts/mktcli.js

```diff
@@ -168,36 +210,38 @@ const main = {
       )
 
     program
-      .command('publication-cancel <coord>')
-      .description('Cancel a publication (x,y)')
+      .command('publication-cancel <assetId> <assetType> <contractName>')
+      .description(
+        'Cancel a publication. `contractName` defaults to `Marketplace`'
+      )
       .action(
-        asSafeAction(async coord => {
-          const [x, y] = parseCLICoords(coord)
-          const tokenId = await Parcel.encodeTokenId(x, y)
-          const contract = eth.getContract('Marketplace')
+        asSafeAction(
+          async (assetId, assetType, contractName = 'Marketplace') => {
+            const asset = await getAssetFromCLIArgs(assetId, assetType)
+            const contract = eth.getContract(contractName)
 
-          await unlockAccountWithPrompt()
-          const txHash = await contract.cancelOrder(tokenId)
+            await unlockAccountWithPrompt()
+            const txHash = await contract.cancelOrder(asset.token_id)
```

Comment:
This will fail 😞 

Now the method is `function cancelOrder(address nftAddress, uint256 assetId)` we need the `address` of the asset

Reply:
good catch!!